packages/tooltip/src/Tooltip/Tooltip.tsx

```diff
@@ -143,11 +146,17 @@ function Tooltip({
               // Without this the tooltip sometimes opens without a transition. flushSync prevents this state update from automatically batching. Instead updates are made synchronously.
               // https://react.dev/reference/react-dom/flushSync#flushing-updates-for-third-party-integrations
               flushSync(() => {
```

Comment:
Do we still need `flushSync`?

Reply:
flushSync is to support react18. Sometimes, tooltip transitions would not appear, so I was curious if the new popover changes fix that issue. But it's understandable if you don't want to touch it right now.

Reply:
makes sense